### PR TITLE
feat(home): V1 Home page

### PR DESCRIPTION
**Pull request by mwhittet**
This PR adds the first version of the home page.

## Changes
- Inital setup of home page, `Header`, `Footer`, `Layout` & `Meta` components and styling.

## Other improvements
- Remove `target` from the `next.config.js` file as it's been [depreciated](https://nextjs.org/docs/messages/deprecated-target-config).
- Various dependency updates.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,117 +1,13 @@
...
-        </div>
-      </main>
-
-      <footer className={styles.footer}>
-        <div className={styles.footerContent}>
-          <p className={styles.footerCopy}>&copy; Barnes Code Ltd. Company no: 09719405</p>
+    <>
+      <div className={styles.bgImage}>
+        <div className="container">
+          <h2>Creative software engineering</h2>
```
Comment (ttbarnes) [inline pages/index.js]: Can we have unit tests for all pages/components and in the same directory please? E.g `pages/index.test.js`

File: components/Meta.jsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <meta property="og:description" content={DESCRIPTION} />
+
+    <meta property="twitter:card" content="summary_large_image" />
+    <meta property="twitter:url" content={DOMAIN} />
+    <meta property="twitter:title" content={TITLE} />
+    <meta property="twitter:description" content={DESCRIPTION} />
+
+    <link href="/apple-touch-icon.png" rel="apple-touch-icon" sizes="180x180" />
+    <link href="/favicon-32x32.png" rel="icon" sizes="32x32" type="image/png" />
+    <link href="/favicon-16x16.png" rel="icon" sizes="16x16" type="image/png" />
```
Comment (ttbarnes) [inline components/Meta.jsx]: Can we use single quotation marks just so it's aligned?

Comment (ttbarnes) [reply components/Meta.jsx]: Also I don't think we need .jsx extension 😎

**Comment by mwhittet** [comment]
@ttbarnes just pushed more the changes requested above with further tests 👍🏻

**Pull request merged**